static/app/components/sidebar/sidebarItem.tsx

```diff
@@ -83,6 +84,15 @@ export type SidebarItemProps = {
    * Additional badge letting users know a tab is in beta.
    */
   isBeta?: boolean;
+  /**
+   * Is the sidebar collapsed or in mobile view
+   */
+  isFloatingSidebar?: boolean;
+
+  /**
+   * Is this item nested within another item
+   */
+  isNested?: boolean;
```

Comment:
This one confused me, isn't `isNested` hard-coded to `true`? When is this false?

Reply:
Its only hardcoded to true in the nested sidebar items,
https://github.com/getsentry/sentry/pull/66313/files#diff-bc2278bb86eb9f14ffeeefeb2727758abf56c7086e66fde575be0bf0cb2ea6b1R189

i.e if its in an accordion

But you mentioned this before, the accordion can just set this on its own, i think i'll change it to do that instead if possible!

Reply:
done!